manifest.toml

```
@@ -46,10 +46,8 @@ ram.runtime = "50M"
     [resources.apt]
     packages = ["mailutils"]
```

```suggestion
    packages = "mailutils"
```

But adding then `, squid` still does not fix the below mystery, I really don't understand 😵